tn_api_hotel.yml

```diff
@@ -46,7 +46,7 @@ servers:
     description: Pre-Production server
 
 paths:
-    /search/{endpoint}/hotel/:
+    /search/hotels{endpoint}:
```

Comment:
Should this be "/search/hotels/{endpoint}" ?

Reply:
fixed